tensorflow_probability/python/bijectors/ascending_test.py

```diff
@@ -0,0 +1,100 @@
+# Copyright 2018 The TensorFlow Probability Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ============================================================================
+"""Tests for Bijector."""
+
+# Dependency imports
+
+import numpy as np
+import tensorflow.compat.v1 as tf1
+import tensorflow.compat.v2 as tf
+from tensorflow_probability.python import bijectors as tfb
+from tensorflow_probability.python.bijectors import bijector_test_util
+from tensorflow_probability.python.internal import tensorshape_util
+from tensorflow_probability.python.internal import test_util
+
+
+@test_util.test_all_tf_execution_regimes
+class AscendingBijectorTest(test_util.TestCase):
+  """Tests correctness of the Ascending transformation."""
+
+  def setUp(self):
+    self._rng = np.random.RandomState(42)
+
+  def testName(self):
+    ascending = tfb.Ascending()
+    self.assertStartsWith(ascending.name, "ascending")
+
+  def testBijectorVector(self):
+    ordered = tfb.Invert(tfb.Ascending())
```

Comment:
I was admittedly a bit lazy to carefully go through the test and swap all x and y and directions; as Ascending() effectively implements Invert(Ordered()), then Invert(Ascending()) is equivalent to Ordered() and I simply copied the tests from ordered_test.py